test/resources/access-om3-checksums-1-0-0.json

```diff
@@ -0,0 +1,8 @@
+{
+  "schema_version": "1-0-0",
+  "output": {
+    "ocean.stats": [
+      "d2f2f18e0f688c1717b52acf78c0793f"
```

Comment:
This needs to be updated to reflect the changed logic in `extract_checksums`.

Reply:
Done in [1671601](https://github.com/ACCESS-NRI/model-config-tests/pull/19/commits/1671601668c2326961370f8e00cd6629669345bd)